clients/fides-js/src/lib/tcf/constants.ts

```diff
@@ -18,41 +17,46 @@ export const ETHYCA_CMP_ID = 407;
 export const FIDES_SEPARATOR = ",";
 
 export const TCF_KEY_MAP: {
-  cookieKey: TcfModelType;
   experienceKey: keyof TcfExperienceRecords;
-  tcfModelKey?: keyof TCModel;
-  enabledIdsKey?: keyof EnabledIds;
+  tcfModelKey:
+    | "purposeConsents"
+    | "purposeLegitimateInterests"
+    | "specialFeatureOptins"
+    | "vendorConsents"
+    | "vendorLegitimateInterests";
+  enabledIdsKey: keyof EnabledIds;
 }[] = [
   {
-    cookieKey: "purpose_consent_preferences",
     experienceKey: "tcf_purpose_consents",
     tcfModelKey: "purposeConsents",
     enabledIdsKey: "purposesConsent",
   },
   {
-    cookieKey: "purpose_legitimate_interests_preferences",
     experienceKey: "tcf_purpose_legitimate_interests",
     tcfModelKey: "purposeLegitimateInterests",
     enabledIdsKey: "purposesLegint",
   },
   {
-    cookieKey: "special_feature_preferences",
     experienceKey: "tcf_special_features",
     tcfModelKey: "specialFeatureOptins",
     enabledIdsKey: "specialFeatures",
   },
   {
-    cookieKey: "vendor_consent_preferences",
     experienceKey: "tcf_vendor_consents",
     tcfModelKey: "vendorConsents",
     enabledIdsKey: "vendorsConsent",
   },
   {
-    cookieKey: "vendor_legitimate_interests_preferences",
     experienceKey: "tcf_vendor_legitimate_interests",
     tcfModelKey: "vendorLegitimateInterests",
     enabledIdsKey: "vendorsLegint",
   },
+];
+
+export const TCF_COOKIE_KEY_MAP: {
```

Comment:
add a comment, something like: we must store these preferences on the cookie outside of the `fides_string` because they are not supported by the FidesString.

Reply:
updated